acceptance/framework/consul/helm_cluster.go

```diff
@@ -86,6 +86,7 @@ func NewHelmCluster(
 		KubectlOptions: ctx.KubectlOptions(t),
 		Logger:         logger,
 		ExtraArgs:      extraArgs,
+		Version:        cfg.HelmChartVersion,
```

Comment:
Just to double check/understand, what happens if the value of `cfg.HelmChartVersion == ""`?  Does it default to something?